Gulpfile.js

```diff
@@ -337,6 +338,36 @@ gulp.task('test-client-legacy-travis-mobile', ['build'], function () {
 gulp.task('travis', [process.env.GULP_TASK || '']);
 
 //Documentation
+function generateItem (name, url, level) {
+    return ' '.repeat(level * 2) + '* [' + name + '](articles' + url + ')\n';
+}
+
+function generateFolder (tocItems, level) {
+    var res = '';
+
+    for (var item of tocItems) {
+        res += generateItem(item.name ? item.name : item.url, item.url, level);
+
+        if (item.content)
+            res += generateFolder(item.content, level + 1);
+    }
+
+    return res;
+}
+
+function generateReadme (toc) {
+    var tocList = generateFolder(toc, 0);
+
+    return '# Documentation\n\n> This documentation looks much better on the [TestCafe website](https://devexpress.github.io/testcafe/getting-started/).\n\n' + tocList;
```

Comment:
`Much better` not quite good wording, I'd rather say that it's easier to browse it on website due to persistent TOC. Also, this docs may be a subject of change, since we use `master` branch for development this directory may contain documentation for functionality which is not release yet.